src/main/resources/db/migration/V20210322_16__CPO-16-CasePaymentOrdersAUD.sql

```diff
@@ -0,0 +1,39 @@
+CREATE SEQUENCE hibernate_sequence INCREMENT 1 MINVALUE 1
+    MAXVALUE 9223372036854775807
+    START 1
+    CACHE 1;
+
+CREATE TABLE public.revinfo
+(
+    rev      integer not null constraint revinfo_pkey primary key,
+    revtstmp bigint
+);
+
+CREATE TABLE public.case_payment_orders_audit
+(
+    id                    uuid    not null,
+    rev                   integer not null,
+    revtype               smallint,
+    action                varchar(70),
+    action_mod            boolean,
+    case_id               bigint,
+    case_id_mod           boolean,
+    created_by            varchar(70),
+    created_by_mod        boolean,
+    created_timestamp     timestamp,
+    created_timestamp_mod boolean,
```

Comment:
We should update the scope doc to include `created_timestamp_mod` if required

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/450c4f00d76e0d3016b2e00af0cef519922b088b